ledger/acctupdates.go

```diff
@@ -985,8 +981,8 @@ func (au *accountUpdates) lookupLatest(addr basics.Address) (data basics.Account
 		}
 
 		// check for resources modified in the past rounds, in the deltas
-		for _, mr := range au.resources.getForAddress(addr) {
-			if err := addResource(mr.resource.CreatableIndex, rnd, mr.resource); err != nil {
+		for cindex, mr := range au.resources.getForAddress(addr) {
```

Comment:
please name it `cidx` for consistency, `cindex` looks like came from `evalindexer.go`. Naming same things by same names reduces cognitive load while reading the code.

Reply:
done